ci_local.sh

```
@@ -0,0 +1,29 @@
+#!/bin/bash
+# EventGate Local CI Quality Gates
+
+set -eo pipefail
+
+echo "=== EventGate Local CI ==="
+echo ""
+
+echo "[1/4] Black (formatting)..."
+black --check $(git ls-files '*.py') || { echo "Run: black \$(git ls-files '*.py')"; exit 1; }
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)